From 54081859e60b1474665cdd08d2e3775cfe6ae177 Mon Sep 17 00:00:00 2001
From: Sai Sree Kartheek Adivi <s-adivi@ti.com>
Date: Sun, 22 May 2022 02:36:14 +0530
Subject: [PATCH] lib: Rename Linux_armv8_64 to Linux_aarch64

Rename Linux_armv8_64 to Linux_aarch64 since there is an issue with
CMakeLists.txt. It's searching for the library in Linux_aarch64
directory in case of aarch64 instead of armv8_64.

Signed-off-by: Sai Sree Kartheek Adivi <s-adivi@ti.com>
---
 .../libPVRScopeDeveloper.a                          | Bin
 1 file changed, 0 insertions(+), 0 deletions(-)
 rename lib/{Linux_armv8_64 => Linux_aarch64}/libPVRScopeDeveloper.a (100%)

diff --git a/lib/Linux_armv8_64/libPVRScopeDeveloper.a b/lib/Linux_aarch64/libPVRScopeDeveloper.a
similarity index 100%
rename from lib/Linux_armv8_64/libPVRScopeDeveloper.a
rename to lib/Linux_aarch64/libPVRScopeDeveloper.a
-- 
2.17.1

